build.gradle

```diff
@@ -66,7 +66,9 @@ buildscript {
     }
 }
 
-
+plugins {
+    id 'org.jetbrains.dokka' version '1.8.20'
```

Comment:
For the version you can use `kotlin_version`

Reply:
According to [Dokka](https://github.com/Kotlin/dokka), the stable release is `1.8.20`. Maybe later, we can change the dependencies system to version catlaogs.